docs/resources/control-flow/user-interactivity/forms/form-widgets/dropdown.md

```diff
@@ -0,0 +1,310 @@
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+
+### Changing dropdown size
+
+To change the height and width of the dropdown, select the **DropDown** widget, move to the **Properties Panel > DropDown Properties > enter the Width and Height value**.
+
+### Set max height
+
+If needed, you can also control the dropdown height using the **Max Height** property.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.loom.com/embed/06fb6dfef8304bb5b3a2d1be4e8fe6b0?sid=80820c77-9ea4-4024-8af2-df09ce978f5c"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Adding margin
+
+Margin adds a space between the DropDown's text and its border. To change the margin, select the **DropDown** widget, move to the **Properties Panel > DropDown Properties >** find the **Margin** property, and change the values.
+
+![Adding margin](../../imgs/adding-margin.gif)
+
+### Changing background color
+
+To change the background color, move to the **Properties Panel > DropDown Style > set the Fill Color**.
+
+![Changing background color](../../imgs/changing-background-color.png)
+
+### Changing menu elevation
+
+Menu elevation adds a shadow to the dropdown, giving it a sense of depth and making it appear above the surface it is placed on.
+
+To change the menu elevation (depth or Z-axis), move the **Properties Panel >** enter the **Menu** **Elevation** value.
+
+:::info
+The higher value draws the bigger size of the shadow.
+:::
+
+### Adding border
+
+See how to [add a border](#).
+
+### Show/hide underline
+
+To show or hide the dropdown underline, move the **Properties Panel >** **DropDown Style** > use the **Hides Underline** toggle.
+
+![Show hide underline](../../imgs/show-hide-underline.gif)
+
+### Fix position
+
+By default, the dropdown options are displayed over/above the dropdown button. To display beneath/below the button, move the **Properties Panel >** **DropDown Style** > switch on the **Fix Position** toggle.
+
+![Fix position for dropdown options](../../imgs/fix-position.webp)
+
+### Trigger action on change
```

Comment:
Add such references in a info admonition. If there are more than one such back references, put them in a list like 
:::info[More properties]
* one
* two
:::

Reply:
Done!